### ISSUE TITLE
Attempted to move window to another screen, window became tiny and can't fix.

### ISSUE THREAD

**Issue by claryfeyart** [camalot/DixperHostWindow#1]
I was attempting to use windows key shortcuts to move it, like the alt-enter to bring up the window to drag, but was unsuccessful. I ended up folding and just picking a different monitor to be my main screen. However, after reloading the host window, it now looks like this.

It's still functional, and I can see little movements from the alerts being played, but I can't resize it no matter what I've attempted. I've uninstalled / restarted / reinstalled, everything. There's also no override in its files that I know of to set a custom window size. 

Here are some screenshots - 
the window's current appearance
https://cdn.discordapp.com/attachments/387861176102420490/908045075232215070/unknown.png
the window playing a tetris affect
https://cdn.discordapp.com/attachments/387861176102420490/908045406338953286/unknown.png
inability to maximize or size the window
https://cdn.discordapp.com/attachments/387861176102420490/908045458239270942/unknown.png

thank you !
ETA: I did uninstall and delete this completely, and deleted any temp files i had associated with it, and it didn't fix my issue. I still don't know how it booted correctly the first time and now refuses to.

**Comment by camalot** [comment]
I have never tried to move it to another monitor. it is set to run on the "main" monitor. it is also only supposed to run "borderless fullscreen". 

I will have to try to reproduce this, and see if there is anything that I can do. 

I have no idea why it isn't resetting to "normal" for you. I don't have anything in there that would save these settings. maybe something part of unity that is doing that by default.

**Comment by claryfeyart** [comment]
> 
> 
> I have never tried to move it to another monitor. it is set to run on the "main" monitor. it is also only supposed to run "borderless fullscreen".
> 
> I will have to try to reproduce this, and see if there is anything that I can do.
> 
> I have no idea why it isn't resetting to "normal" for you. I don't have anything in there that would save these settings. maybe something part of unity that is doing that by default.

Yeah, I couldn't move it so I left it on my main monitor, but changed my settings to have a different monitor be my main, thinking that it would be fine and it booted up like this. 

Unity is a possibility I didn't consider, though I'm too unfamiliar with it to do anything myself. The only other option I thought of was that it stored data in the registry but that felt a bit farfetched lol

If you'd like to, I can screenshare through discord or something similar if you'd like to inspect what i've got going on if you can't replicate this issue. ClaryFeyArt#2818 is my handle there

**Comment by claryfeyart** [comment]
Small update, did a clean wipe of all dixper files and unity files and these here, and tried this again. I opened one window after another, and noticed they were getting bigger. 

Then, i tried to cascade them for a screenshot and they all shrank to the tiny size again. so, i think thats what happened? it's my only idea. I suppose i'll just open up enough of these that they get large enough to see out of, until there's some fix or something.

https://cdn.discordapp.com/attachments/704023170289762315/908161168189825085/unknown.png

this biggest one here, disappeared like the original window should, as soon as the logo loaded. 
https://cdn.discordapp.com/attachments/876658817356685332/908162785060470814/unknown.png

then it immediately started as small as possible again.
